src/cli/flags/m365_common.go

```diff
@@ -39,4 +40,36 @@ func AddDataFlag(cmd *cobra.Command, allowed []string, hide bool) {
 	if hide {
 		cobra.CheckErr(fs.MarkHidden(CategoryDataFN))
 	}
+
+	// TODO(meain): This is a hacky way to get it to autocomplete multiple items
+	cobra.CheckErr(cmd.RegisterFlagCompletionFunc(
```

Comment:
Could we move the multiple value related changes to a separate PR, just to limit the complexity of this PR?